src/usps.js

```diff
@@ -259,7 +253,7 @@ function callUSPS(api, method, property, config, params, callback) {
         though it may actually have arrays, so returning first cell
       */
 
-      var specificResult = {};
+      const specificResult = {};
       if (result && result[responseName] && result[responseName][property]) {
```

Comment:
can not wait for [optional chaining](https://github.com/tc39/proposal-optional-chaining#stacking) to land JS...

Reply:
That looks delicious.